core-web/libs/sdk/angular/src/lib/layout/dotcms-layout/dotcms-layout.component.ts

```diff
@@ -52,21 +77,22 @@ export class DotcmsLayoutComponent implements OnInit {
             )
             .subscribe((urlSegments) => {
                 const pathname = '/' + urlSegments.join('/');
-                const config = {
-                    pathname,
-                    onReload: () => {
-                        // Reload the page when the user edit the page
-                        this.router.navigate([pathname], {
-                            onSameUrlNavigation: 'reload' // Force Angular to reload the page
-                        });
-                    }
-                };
-                initEditor(config);
+
+                initEditor();
                 updateNavigation(pathname || '/');
+
+                //Sent the path to the editor
+                postMessageToEditor({
+                    action: CUSTOMER_ACTIONS.GET_PAGE_INFO,
+                    payload: {
+                        pathname
+                    }
+                });
             });
     }
 
     ngOnChanges() {
+        this.pageAssetData.set(this.pageAsset);
```

Comment:
If I am listing for the window message in the host listener, why do I need to watch changes here as well?

Reply:
I changed the logic here to avoid the double variable and now we dont need this line, removed!